src/rpcrequest-payload.ts

```diff
@@ -65,7 +65,7 @@ class RpcRequestPayload implements Iterable<Buffer> {
   }
 
   * generateParameterData(parameter: Parameter, options: any) {
-    const buffer = new WritableTrackingBuffer(0);
+    const buffer = new WritableTrackingBuffer(1 + 2 + Buffer.byteLength(parameter.name, 'ucs-2') + 1);
```

Comment:
This ensures the `Buffer` used by `WritableTrackingBuffer` is big enough to contain the data that will be written to it without causing any resizing,